backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -437,9 +466,21 @@ private async Task<List<Rule>> DeleteAllRulesInPolicy(RequestToDelete policyToDe
                 return null;
             }
 
+            if (resourceMatchType == ResourceAttributeMatchType.AltinnApp)
+            {
+                altinnAppId = $"{org}/{app}";
+            }
+
+            resource = await _resourceRegistryClient.GetResource(resourceRegistryId);
```

Comment:
same here, use resourceMatchType.ResourceRegistry for specific logic as _resourceRegistryClient should not be used for altinn app scenario